docker-entrypoint.sh

```diff
@@ -47,4 +47,15 @@ EOT
     fi
 fi
 
+UPLOAD_LIMIT_INI_FILE="$PHP_INI_DIR/conf.d/phpmyadmin-upload-limit.ini"
+if [ ! -z "${UPLOAD_LIMIT}" ]; then
+    echo "Adding the custom upload limit."
+    echo -e "upload_max_filesize = $UPLOAD_LIMIT\npost_max_size = $UPLOAD_LIMIT\n" > $UPLOAD_LIMIT_INI_FILE
+else
+    if [ -f $UPLOAD_LIMIT_INI_FILE ]; then
```

Comment:
In which case is `$UPLOAD_LIMIT_INI_FILE` present without specifying a `$UPLOAD_LIMIT`?

Reply:
I need to test by maybe with docker compose when changing the value

Reply:
c4af17e703dfcce8b698cf6b6c90f287bdcc3f0e